apolloschurchapp/src/index.js

```diff
@@ -26,34 +23,19 @@ const AppStatusBar = withTheme(({ theme }) => ({
   backgroundColor: theme.colors.paper,
 }))(StatusBar);
 
-const ProtectedRouteWithSplashScreen = (props) => {
-  const handleOnRouteChange = () => SplashScreen.hide();
-
-  return <ProtectedRoute {...props} onRouteChange={handleOnRouteChange} />;
-};
-
-// Hack to avoid needing to pass emailRequired through the navigator.navigate
-const EnhancedAuth = (props) => <Auth {...props} emailRequired />;
-// 😑
-hoistNonReactStatic(EnhancedAuth, Auth);
-
 const AppNavigator = createStackNavigator(
   {
-    ProtectedRoute: ProtectedRouteWithSplashScreen,
     Tabs,
     ContentSingle,
     Event,
-    Auth: EnhancedAuth,
-    PersonalDetails,
-    ChangePassword,
     Location,
     Passes,
     UserWebBrowser,
     Onboarding,
     LandingScreen,
   },
   {
-    initialRouteName: 'ProtectedRoute',
+    initialRouteName: 'LandingScreen',
```

Comment:
Something to think about for future - every time the app closes and re-opens it'll open the landing screen. We're probably going to need to store some value in app storage on - has user seen onboarding? - and push them to the `Tabs` screen if they have.